### PR TITLE
Address quick-start.md issues

### PR DESCRIPTION
**Pull request by bradbeck**
Addresses the following issues:
* https://github.com/thesecuresoftwarefactory/ssf/issues/99
* https://github.com/thesecuresoftwarefactory/ssf/issues/100

### CODE REVIEW

**Review by joshualucas84 — APPROVED** [review]
Looks good!

File: docs/content/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -77,22 +77,24 @@ tkn pr logs --last -f
...
 We start by defining some variables to simplify the validation commands:
 
 ```bash
-IMAGE_URL=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageUrl") | .value')
+IMAGE_URL=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="imageUrl")].value}')
 export IMAGE_URL=localhost:8888${IMAGE_URL#"registry.kube-system.svc.cluster.local"}
-export IMAGE_TAG=$(tkn pr describe --last -o json | jq -r '.spec.params[] | select(.name=="imageTag") | .value')
+export IMAGE_TAG=$(tkn pr describe --last -o jsonpath='{.spec.params[?(@.name=="imageTag")].value}')
 export DOCKER_IMG="${IMAGE_URL}:${IMAGE_TAG}"
-export COSIGN_KEY="k8s://tekton-chains/signing-secrets"
```
Comment (rgreinho) [inline docs/content/docs/getting-started/quick-start.md:89]: Why removing this value? Since it used several times, it makes sense to have it no?

Comment (bradbeck) [reply docs/content/docs/getting-started/quick-start.md:89]: Changed it to match what is currently in https://github.com/thesecuresoftwarefactory/ssf/blob/main/examples/ibm-tutorial/README.md

Comment (rgreinho) [reply docs/content/docs/getting-started/quick-start.md:89]: Can update the README.md instead then?

Comment (bradbeck) [reply docs/content/docs/getting-started/quick-start.md:89]: I guess I'm inclined to have fewer variables to carry around in the examples users are expected to copy and paste. The other variables are holding values that are derived at runtime whereas `COSIGN_KEY` is a static value.

Comment (rgreinho) [reply docs/content/docs/getting-started/quick-start.md:89]: oh ok... I'm not sure I agree, and I think the opposite is more correct, but that should not be a blocker.

**Review by rgreinho — APPROVED** [review]
👍

**Review by rgreinho — APPROVED** [review]

**Review by rgreinho — APPROVED** [review]

**Review by pxp928 — APPROVED** [review]

**Pull request merged**